### ISSUE TITLE
Month and day names are not translated to Welsh (locale: cy) on Chrome on Windows

### ISSUE THREAD

**Issue by LeaTark** [fullcalendar/fullcalendar#5565]
How can month and day name translations be provided in v5?

monthNames, monthNamesShort, dayNames and dayNamesShort are not recognised in locale.

Thank you

**Comment by acerix** [comment]
Would you be able to post a [runnable, stripped-down demonstration of the bug](http://fullcalendar.io/wiki/Reduced-Test-Cases/)? Would really appreciate it because the time saved reproducing will be time spent fixing.

The translations should be automatic as in this demo:

https://fullcalendar.io/docs/locale-demo

**Comment by LeaTark** [comment]
Why ask for a demo and close the issue?
Here's a pen https://codepen.io/LeaTark/pen/eYJypqo
Could you re-open please?

The button text is translated but the day and month names are not.

**Comment by acerix** [comment]
Thanks for the demo, but I'm seeing that text as translated, for example I see the month as "Gorffennaf", is that not correct? The day names also are translated, "Sul, Llun, etc.".  If you don't see that, let me know which browser.

The month names are defined by the browser, if you need more control over the format, you can use the moment plugin.

**Comment by LeaTark** [comment]
That would be correct but it's not what I see.

The button text is translated but month and day names remain in English

![image](https://user-images.githubusercontent.com/25925693/86447662-96c5ee80-bd0d-11ea-9077-cdd0c4ba4599.png)

If I switch to French, for example, then they are translated

![image](https://user-images.githubusercontent.com/25925693/86447551-6e3df480-bd0d-11ea-8a48-48834431bee1.png)

I'm using Chrome version 80.0.3987.132 on Windows 10

Thanks

**Comment by acerix** [comment]
I see the translations in Chromium 83 on Linux, so maybe they are only missing on Windows, or were added since version 80.  So you would need the moment plugin to get them working in that browser, that is explained here:

https://fullcalendar.io/docs/moment-plugins

**Comment by iiibbbmmm** [comment]
Both view title and column header(in this demo) used `NativeFormatter` for Date formatting.
It depends on `Intl.DateTimeFormat`,  and Chrome on Win10, even 83,  do not support `cy` locale.
```javascript
Intl.DateTimeFormat.supportedLocalesOf("cy");
```
should tell the difference, on Firefox 76, it returns `["cy"]`, but on Chrome win10 83, it returns `[]`.

In this case, custom locale settings were not involving at all , that's the real problem.

**Comment by JohnRDOrazio** [comment]
I'm having some difficulty because I need to render day names and month names in Latin. However the browser `Intl` object does not include the Latin language (not really a priority for browsers seeing that it's a classical non spoken language).

It would be nice if the `dayNames`, `dayNamesShort`, `monthNames` and `monthNamesShort` were still available in FullCalendar 5 as a last resort.

**Comment by arshaw** [comment]
@LeaTark and @JohnRDOrazio, for both of your issues, this ticket, when implemented, will be a solution:
https://github.com/fullcalendar/fullcalendar/issues/5683
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?

TL;DR: FullCalendar should be modified to lean on the moment/luxon formatting plugins when the browser does not fully support a given locale.

**Comment by she7ata** [comment]
Dears, I wanted also to change the month names or use a custom ones, not the language, properly as the request above, is there is any news about it or a way to give an array of monthNames ?

**Comment by codebymikey** [comment]
Related issue: https://github.com/formatjs/formatjs/issues/3245
